recipes/starshot_accessibility/recipe.yml

```diff
@@ -0,0 +1,24 @@
+name: Accessibility
+type: Starshot
+description: 'Sets up accessibility tools and functionality.'
+recipes:
+  - starshot_admin_theme
+install:
+  - a11y_form_helpers
+  - block_aria_landmark_roles
+  - editoria11y
```

Comment:
Same here -- there's a separate issue already for adding this module (#82), so if it's not strictly necessary for this recipe, maybe it'd be best to remove it.

Reply:
I added a new recipe for starshot_editoria11y but left it as optional like the multilingual. I can remove it and add this in another MR if necessary.